editor/sidebar/post-format/index.js

```diff
@@ -2,91 +2,38 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
-import { get, find, flowRight } from 'lodash';
+import { get, flowRight } from 'lodash';
 
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
-import { PanelRow, withAPIData, withInstanceId } from '@wordpress/components';
+import { PanelRow, withAPIData } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import './style.scss';
-import {
-	getEditedPostAttribute,
-	getSuggestedPostFormat,
-	getCurrentPostType,
-} from '../../selectors';
-import { editPost } from '../../actions';
+import PostFormatSelector from '../../post-format';
+import { getCurrentPostType } from '../../selectors';
 
-const POST_FORMATS = [
-	{ id: 'aside', caption: __( 'Aside' ) },
-	{ id: 'gallery', caption: __( 'Gallery' ) },
-	{ id: 'link', caption: __( 'Link' ) },
-	{ id: 'image', caption: __( 'Image' ) },
-	{ id: 'quote', caption: __( 'Quote' ) },
-	{ id: 'standard', caption: __( 'Standard' ) },
-	{ id: 'status', caption: __( 'Status' ) },
-	{ id: 'video', caption: __( 'Video' ) },
-	{ id: 'audio', caption: __( 'Audio' ) },
-	{ id: 'chat', caption: __( 'Chat' ) },
-];
-
-function PostFormat( { postType, onUpdatePostFormat, postFormat = 'standard', suggestedFormat, instanceId } ) {
+function PostFormat( { postType } ) {
 	if ( ! get( postType.data, [ 'supports', 'post-formats' ] ) ) {
```

Comment:
I hesitated whether to keep this in the reusable component. Even if we do so, we'd have to duplicate it here to avoid rendering the container panel as well. Thoughts @aduth ?

Reply:
Hm, if the reusable component is still "post aware" it seems to make sense to also be post post-type aware, and reasonable enough to render nothing if the post type does not support. That it's duplicated seems okay.

Reply:
Is it worth applying the same principle as with `PostStickyCheck` in #3114?